File: backend/src/main/kotlin/com/ramap/model/Review.kt
package com.ramap.model

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import java.time.LocalDateTime

// 리뷰 MongoDB 도큐먼트
@Document(collection = "reviews")
data class Review(
    @Id
    val id: String? = null,
    val shopId: String,
    val authorNickname: String,
    val rating: Int,
    val content: String,
    val createdAt: LocalDateTime = LocalDateTime.now()
)

File: frontend/src/types/review.ts
// 리뷰 타입 정의
export interface Review {
  id: string
  shopId: string
  authorNickname: string
  rating: number
  content: string
  createdAt: string
}

// 리뷰 작성 요청 타입
export interface ReviewRequest {
  authorNickname: string
  rating: number
  content: string
}
